css-two-column-layout/styles.css

```diff
@@ -0,0 +1,85 @@
+.container {
+  margin: auto;
+  max-width: 1200px;
+  background-color: white;
+}
+
+.row {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.col-full {
+  width: 100%;
+}
+
+.col-half {
+  width: calc(100%/2);
+}
+
+.col-fourth {
+  width: 25%;
+  text-align: right;
+  padding-right: 0;
+}
+
+.col-three-fourths {
+  width: 75%;
+  align-items: right;
+  text-align: left;
+}
+
+.margintop {
+  margin-top: 3rem;
+}
+
+.marginbottom {
+  margin-bottom: 3rem;
+}
+
+.pad {
+  padding: 2rem;
+}
+
+h1 {
+  font-size: 400%;
+  margin-bottom: 0;
+}
+
+h3 {
+  text-transform: uppercase;
+  font-weight: 900;
+  padding-top: 1.5rem;
+}
+
+p {
+  line-height: 150%;
+}
+
+body {
+  font-family: Karla, sans-serif;
+  letter-spacing: 0.5px;
+  background-color: white;
+}
+
+.text-center {
+  text-align: center;
+}
+
+button {
+  background-color: white;
+  border: 0.15rem solid #d45751;
+  color: #d45751;
+  font-family: Karla, sans-serif;
+  text-align: center;
+  margin-top: 3rem;
+  margin-bottom: 25%;
+  width: 250px;
+  height: 50px;
+  font-weight: bolder;
+  font-size: larger;
+}
+
+#learnmore {
+  margin-left: 425px;
```

Comment:
😱 

There is a better way to do this, I'm sure. `text-align` typically is plenty to center a button element